src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -128,6 +128,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 	private _latestXtermWriteData: number = 0;
 	private _latestXtermParseData: number = 0;
 	private _isExiting: boolean;
+	private _enableShellIntegration: boolean = false;
```

Comment:
I think you said the 133;E sequence was custom in which case we would want to use our own OSC code for. But I've been thinking we shouldn't need that at all, we could infer shell integration is on if needed by querying the capabilities it turns on.